src/NUnitEngine/nunit.engine.core/Drivers/NUnitFrameworkApi2009.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Runtime.Serialization;
+using NUnit.Common;
+using NUnit.Engine.Internal;
+
+namespace NUnit.Engine.Drivers
+{
+#if NETFRAMEWORK
```

Comment:
Suggest to move the conditional to the top of the file, excluding the whole file.
Now for non-framework builds the `using` statements are marked as unnecessary.